### PR TITLE
[251] Fee Endpoint - Cypress test changes

### PR DESCRIPTION
**Pull request by W3stside**
### Part of #263 

This PR addresses @alfetopito that the cypress tests were failing

They pass:
Screenshot from 2021-04-06 15-34-27

Updates `commands.test.js` to add 2 helper methods and update some of the logic
Updates `fee.test.ts` to make pass

### CODE REVIEW

File: cypress-custom/integration/fee.test.ts [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 const RINKEBY = ChainId.RINKEBY.toString()
-const FEE_QUERY = `https://protocol-rinkeby.dev.gnosisdev.com/api/v1/tokens/${WETH[4].address}/fee`
 const FEE_QUOTES_LOCAL_STORAGE_KEY = 'redux_localstorage_simple_fee'
 const FOUR_HOURS = 3600 * 4 * 1000
 const DEFAULT_SELL_TOKEN = WETH[ChainId.RINKEBY].address
 
-function _assertFeeData(fee: FeeInformation): void {
-  expect(fee).to.have.property('minimalFee')
-  expect(fee).to.have.property('feeRatio')
+const getFeeQuery = ({ sellToken, buyToken, amount, kind }: any) =>
```
Comment (alfetopito) [inline cypress-custom/integration/fee.test.ts:11]: Nitpick: why not typed?

File: cypress-custom/integration/fee.test.ts [hunk 2]
```diff
@@ -45,8 +50,17 @@ function _assertFeeFetched(token: string): Cypress.Chainable {
...
 }
 
+console.log('[function]::', _assertFeeFetched)
+
 describe('Fee endpoint', () => {
   it('Returns the expected info', () => {
+    const FEE_QUERY = getFeeQuery({
+      sellToken: DEFAULT_SELL_TOKEN,
+      buyToken: DAI,
+      amount: 0.1 * 10 ** 18 + '',
```
Comment (alfetopito) [inline cypress-custom/integration/fee.test.ts:60]: Nitpick: constant? function?

**Review by alfetopito — APPROVED** [review]
Looks good, tests passing. Only minor nitpicks.

File: cypress-custom/integration/fee.test.ts (hunk 1 above)
Comment (W3stside) [reply cypress-custom/integration/fee.test.ts:11]: pure laziness, it's pretty straight forward implementation but i can change this

File: cypress-custom/integration/fee.test.ts (hunk 2 above)
Comment (W3stside) [reply cypress-custom/integration/fee.test.ts:60]: :cry: okidoki

File: cypress-custom/integration/fee.test.ts [hunk 3]
```diff
@@ -98,23 +122,36 @@ describe('Fee: Complex fetch and persist fee', () => {
...
+    buyToken: DAI,
+    amount: parseUnits(INPUT_AMOUNT, DEFAULT_SELL_TOKEN.decimals).toString(),
+    kind: OrderKind.SELL
   })
-
-  it('Fetch fee when selecting token', () => {
+  const FEE_RESP = {
+    // 1 min in future
+    expirationDate: new Date(Date.now() + 60000).toISOString(),
+    amount: (0.05 * 10 ** 18).toString()
```
Comment (alfetopito) [inline cypress-custom/integration/fee.test.ts:135]: Forgot one :)

```suggestion
    amount: parseUnits('0.05', DEFAULT_SELL_TOKEN.decimals).toString()
```

**Pull request merged**